### ISSUE TITLE
Bug in the variational quantum classifier example

### ISSUE THREAD

**Issue by mlazzarin** [Quantum-TII/qibo#367]
Hi! I was looking at the example with the variational classifier (thanks for the useful tutorial!) and I found a line of code which looks wrong to me. In particular, the training set is defined in these lines:
https://github.com/Quantum-TII/qibo/blob/1d27fb2c49d5b7e992eee65f6994e7b60acdc7ee/examples/variational_classifier/main.py#L43-L45
while the test set is defined in these ones:
https://github.com/Quantum-TII/qibo/blob/1d27fb2c49d5b7e992eee65f6994e7b60acdc7ee/examples/variational_classifier/main.py#L79-L81
I think that `data[49]`, `data[99]` and `data[149]` are not included in the slices in line 80, so `data_test` ends up with 42 elements while `labels_test` has 45 elements. The accuracy is computed with zip, which works anyway, but labels and predictors are not aligned anymore so the test accuracy (e.g. the 67% written in the corresponding README.md) is underestimated.

**Comment by scarrazza** [comment]
@mlazzarin thanks for reporting this issue. 
@DiegoGM91 could you please confirm if the interpretation is correct?

**Comment by DiegoGM91** [comment]
@mlazzarin thanks for spotting this. We are happy to hear that you find the tutorials useful.
I think you are completely right, so I will open a PR to fix it @scarrazza.
